### PR TITLE
build(app): CI

### PR DESCRIPTION
**Pull request by juskek**

### CODE REVIEW

File: front/src/app/__tests__/page.test.tsx
Comment (Lerri-Cofannos) [inline front/src/app/__tests__/page.test.tsx:1]: Should this test be merged to main? Was it not only to test whether the CI ran when opening a PR?

File: .github/workflows/app-ci.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      - name: Checkout main
+        uses: actions/checkout@v3
+      - name: Setup Node 18
+        uses: actions/setup-node@v3
+        with: 
+          node-version: 18
+      - name: Create env file
+        run: |
+          pwd
+          echo "${{ secrets.FRONT_ENV_FILE }}" > .env
```
Comment (CallumHemsley) [inline .github/workflows/app-ci.yml:28]: Required for CI?

**Review by Lerri-Cofannos — COMMENTED** [review]
Nice one! Two small questions

File: front/src/app/__tests__/page.test.tsx
Comment (juskek) [reply front/src/app/__tests__/page.test.tsx:1]: jest fails if there are no tests found - this test file will be replaced with proper tests in future tickets

**Review by CallumHemsley — APPROVED** [review]

**Pull request merged**